zlib-ng.map

```diff
@@ -64,17 +64,14 @@ ZLIB_NG_2.0.0 {
     zng_uncompress2;
     zng_zError;
     zng_zlibCompileFlags;
-    zng_zlibng_string;
-    zng_version;
+    zng_vstring;
   local:
     zng_deflate_copyright;
     zng_inflate_copyright;
-    zng_inflate_fast;
-    zng_inflate_table;
-    zng_zcalloc;
-    zng_zcfree;
+    zng_calloc;
+    zng_cfree;
```

Comment:
These lines don't look correct.

Reply:
Good catch.

Reply:
Fix pushed.

Reply:
I'm usually good at catching if there is something that conflicts with changes in other parts of the code. Kind of helps having 4K monitor with really small font size ;)

Reply:
I renamed the calloc and cfree functions everywhere 3 times before I landed on the current names that I figured were the best suited for both modes. This change must have slipped through the cracks when making the fixup patch and rebasing, and that is why PR reviews are so important.

Reply:
That's why I usually annoy certain person if I want someone to double-check my changes... Each one of us is good at looking specific issues when reviewing PRs.